### ISSUE TITLE
Breaking change in Django-Storages

### ISSUE THREAD

**Issue by stuartmaxwell** [testdrivenio/django-docker-s3#7]
Hi there - thanks for writing up this repo and the accompanying article. Just wanted to let you know that there have been a number of breaking changes with the latest update of Django-Storages. The result is that files uploaded with the `collectstatic` files will not be set to `public-read` permissions, which obviously means that all the static files are not accessible to the public.

The fix is easy and you need to replace:
```python
AWS_DEFAULT_ACL = 'public-read'
```
with
```python
AWS_S3_OBJECT_PARAMETERS = {
    'ACL': 'public-read',
}
```

Also, because your code is also changing the `CacheControl` parameter, the full fix is as follows:
```python
AWS_S3_OBJECT_PARAMETERS = {
    "ACL": "public-read",
    "CacheControl": "max-age=86400"
}
```

Hope that helps - this issue has more details and better explanation: https://github.com/jschneier/django-storages/issues/927

Cheers - Stuart.

**Comment by mjhea0** [comment]
Thanks for sharing this @stuartmaxwell. I'll make the changes next time I update the post.

**Comment by readybeginn** [comment]
This was fixed in release 1.10.1 per [this comment.](https://github.com/jschneier/django-storages/issues/927#issuecomment-691837541)

**Comment by mjhea0** [comment]
Updating the post now. Using version 1.11.1 and it still works fine with `AWS_DEFAULT_ACL / default_acl`
